src/libraries/Microsoft.PowerFx.LanguageServerProtocol/LanguageServer/LanguageServer.cs

```diff
@@ -35,6 +35,11 @@ public class LanguageServer
     {
         private const char EOL = '\n';
 
+        /// <summary>
+        /// This const represents the dummy formula that is used to create an infrastructure needed to get the symbols for Nl2Fx operation.
+        /// </summary>
+        internal static string Nl2FxDummyFormula = '"' + Guid.NewGuid().ToString() + '"';
```

Comment:
>Guid.NewGuid().ToString() [](http://example.com/codeflow?start=57&length=25)

Probably ok for this to be a fixed guid value. 
That may make debugging easier too.

Reply:
Using a static GUID had been my interpretation of this as well. That lets us search code for a very specific string to see if they use this.

Reply:
Okay are you both suggesting to generate a GUID let's say "blahblah" and that becomes our dummy formula for eternity?

Reply:
Correct (although not for eternity, just for however long this dummy formula code lives for.)

Reply:
I guess it would be there all the time now. When i come up with design with more hooks which involves one of the "pre nl2fx" hook during which this dummy formula is used, we would still need to provide the default hook implementation which is just going to be running Check on this dummy formula

Reply:
Done